doc/classes/CollisionObject2D.xml

```diff
@@ -19,7 +19,7 @@
 			<argument index="2" name="shape_idx" type="int">
 			</argument>
 			<description>
-				Accepts unhandled [InputEvent]s. [code]shape_idx[/code] is the child index of the clicked [Shape2D]. Connect to the [code]input_event[/code] signal to easily pick up these events.
+				Accepts unhandled [InputEvent]s. Requires [code]input_pickable[/code] to be [code]true[/code]. [code]shape_idx[/code] is the child index of the clicked [Shape2D]. Connect to the [code]input_event[/code] signal to easily pick up these events.
```

Comment:
Properties should be referred to with the `[member]` tag to create a hyperlink, here `[member input_pickable]`.

Could you also squash commits together?

Reply:
I took the opportunity to change all occurrences of `[code]input_pickable[/code]` to `[member input_pickable]`.

I hope I did it right